packages/adaptive_scaffold/lib/src/adaptive_layout.dart

```diff
@@ -0,0 +1,513 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+import 'package:flutter/widgets.dart';
+import 'breakpoint.dart';
+import 'slot_layout.dart';
+import 'slot_layout_config.dart';
+
+/// A parent Widget takes in multiple [SlotLayout] components and places them
```

Comment:
I'd take out the "parent".  Also, since this is the main class in the package, I'd be pretty verbose here about how to use it, and what classes are involved, what the terminology is (e.g. what's a "slot", what's a "breakpoint"?).  You might also link to the Material Design spec (if it's public, which I think it is).